wfc_dylib/src/lib.rs

```diff
@@ -311,6 +311,24 @@ pub unsafe extern "C" fn wfc_world_state_slots_set(
     }
 }
 
+/// Gets the current world status without making an observation.
```

Comment:
This doc needs a "# Safety" section regarding the state of the handle. Look at the other functions accepting `WfcWorldStateHandle` for inspiration.